src/main/java/ev3dev/utils/Sysfs2.java

```diff
@@ -166,6 +166,7 @@ public static boolean writeBytes(final String path, final byte[] value) {
     //}
 
     //Static Setter to test the idea from @dwalend
+/*
```

Comment:
I commented out this big block of stuff that's not used anymore. OK to delete it?

Reply:
Yes please, remove it.